cmake/modules/plugin-sdk-cpp.cmake

```diff
@@ -8,7 +8,7 @@ message(
 FetchContent_Declare(
   plugin-sdk-cpp
   GIT_REPOSITORY https://github.com/falcosecurity/plugin-sdk-cpp.git
-  GIT_TAG 0.2.0)
+  GIT_TAG e59ae27d443ea37629c72a4a4e75a39740bbc4ed)
```

Comment:
TODO: release 0.2.1 of the plugin sdk once https://github.com/falcosecurity/plugin-sdk-cpp/pull/54 is merged.